### PR TITLE
Upload-Flow patch for not installing hidden packs

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://panw-global.slack.com/archives/G011E63JXPB/p1612220331171100

## Description
Added a patch to not install hidden packs on Upload-Flow

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by guyfreund** [comment]
Successful run:

CCI: https://app.circleci.com/pipelines/github/demisto/content/65123/workflows/a46cc0da-8591-4361-bfd9-0c41a757ec1d
Bucket: https://console.cloud.google.com/storage/browser/marketplace-ci-build/content/builds/upload-install-bugfix/268877
Server Configuration: https://storage.googleapis.com/marketplace-ci-build/content/builds/upload-install-bugfix/268877

**Comment by guyfreund** [comment]
Successful run:

CCI: https://app.circleci.com/pipelines/github/demisto/content/65123/workflows/a46cc0da-8591-4361-bfd9-0c41a757ec1d
Bucket: https://console.cloud.google.com/storage/browser/marketplace-ci-build/content/builds/upload-install-bugfix/268877
Server Configuration: https://storage.googleapis.com/marketplace-ci-build/content/builds/upload-install-bugfix/268877

**Comment by ShahafBenYakir** [comment]
Nice.
NIghtly upload failure:
https:::circle-production-customer-artifacts s3 am

This PR run:
https:::circle-production-customer-artifacts s3 am2

Can be seen that the `Skyformation` deprecated pack was not uploaded in this fix PR and the upload flow succeeded.
FYI @yuvalbenshalom @guyfreund

File: Tests/Marketplace/search_and_install_packs.py [hunk 1]
```diff
@@ -474,8 +474,9 @@ def install_all_content_packs(client: demisto_client, host: str, server_version:
                 pack_metadata = json.load(json_file)
                 pack_version = pack_metadata.get('currentVersion')
                 server_min_version = pack_metadata.get('serverMinVersion', '6.0.0')
+                hidden = pack_metadata.get('hidden', False)
             # Check if the server version is greater than the minimum server version required for this pack:
```
Comment (ShahafBenYakir) [inline Tests/Marketplace/search_and_install_packs.py:478]:
```suggestion
            # Check if the server version is greater than the minimum server version required for this pack or if the pack is hidden (deprecated):
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**